Models/DTO/Administrator/FilterPharmaciesReportDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.DTO.Administrator
+{
+    public class FilterPharmaciesReportDTO
+    {
+        [Required]
+        [DataType(DataType.DateTime)]
+        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
+        [JsonProperty("datefrom")]
+        public DateTime DateFrom { get; set; }
+
+        [Required]
+        [DataType(DataType.DateTime)]
+        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
+        [JsonProperty("dateto")]
```

Comment:
Maybe it would be more readable to serialize as `date-to`?

Reply:
Changed

Reply:
I agree

Reply:
We can use just letters no special symbols.

Reply:
We can use only letters without special symbols.

Reply:
What about `dateTo`?

Reply:
If we want use uppercases than we will have to change all models. where is json annotanions.